src/tests/functional/plugin/shared/src/snippets/group_normalization.cpp

```diff
@@ -0,0 +1,73 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "common_test_utils/common_utils.hpp"
+#include "snippets/group_normalization.hpp"
+#include "subgraph_group_normalization.hpp"
+#include "functional_test_utils/skip_tests_config.hpp"
+
+namespace ov {
+namespace test {
+namespace snippets {
+
+std::string GroupNormalization::getTestCaseName(testing::TestParamInfo<ov::test::snippets::GroupNormalizationParams> obj) {
+    InputShape inputShapes;
+    size_t numGroup;
+    float eps;
+    std::string targetDevice;
+    size_t num_nodes, num_subgraphs;
+    std::tie(inputShapes, numGroup, eps, num_nodes, num_subgraphs, targetDevice) = obj.param;
+
+    std::ostringstream result;
+    result << "IS=" << ov::test::utils::partialShape2str({inputShapes.first}) << "_";
+    result << "TS=";
+    for (const auto& shape : inputShapes.second) {
+        result << "(" << ov::test::utils::vec2str(shape) << ")_";
+    }
+    result << "numGroup=" << numGroup << "_";
+    result << "epsilon=" << eps << "_";
+    result << "#N=" << num_nodes << "_";
+    result << "#S=" << num_subgraphs << "_";
+    result << "targetDevice=" << targetDevice;
+    return result.str();
+}
+
+void GroupNormalization::SetUp() {
+    InputShape inputShape;
+    size_t numGroup;
+    float eps;
+    std::tie(inputShape, numGroup, eps, ref_num_nodes, ref_num_subgraphs, targetDevice) = this->GetParam();
+
+    InputShape scaleShiftShape = ExtractScaleShiftShape(inputShape);
+
+    init_input_shapes({inputShape, scaleShiftShape, scaleShiftShape});
+
+    auto f = ov::test::snippets::GroupNormalizationFunction(inputDynamicShapes, numGroup, eps);
+    function = f.getOriginal();
+
+    if (!configuration.count("SNIPPETS_MODE")) {
+        configuration.insert({"SNIPPETS_MODE", "IGNORE_CALLBACK"});
+    }
```

Comment:
You have already disabled `callback`! As I noticed before, you can just add `if not IGNORE_CALLBACK` in the corresponding callback transformation pipeline to check more input parameters 😃

Reply:
done, Thanks Alexandra!